src/DynamoCLI/CommandLineRunner.cs

```diff
@@ -44,57 +43,18 @@ private static XmlDocument RunCommandLineArgs(DynamoModel model, StartupUtils.Co
             Console.WriteLine("loaded file");
             model.EvaluationCompleted += (o, args) => { evalComplete = true; };
 
-            if (!string.IsNullOrEmpty(cmdLineArgs.PresetFilePath))
-            {
-                //first load the openfile nodegraph
-                var originalGraphdoc = XmlHelper.CreateDocument("tempworkspace");
-                originalGraphdoc.Load(cmdLineArgs.OpenFilePath);
-                var graph = NodeGraph.LoadGraphFromXml(originalGraphdoc, model.NodeFactory);
-
-                //then load the presetsfile nodegraph (this should only contain presets),
-                var presetsDoc = XmlHelper.CreateDocument("presetstempworkspace");
-                presetsDoc.Load(cmdLineArgs.PresetFilePath);
-                //when we load the presets we need to pass in the nodeModels from the original graph
-                var presets = NodeGraph.LoadPresetsFromXml(presetsDoc, graph.Nodes);
-
-                //load the presets contained in the presetsfile into the workspace,
-                model.CurrentWorkspace.ImportPresets(presets);
-            }
-
-            //build a list of states, for now, none, a single state, or all of them
-            //this must be done after potentially loading states from external file
+            // Build a list of states, by default there is only a single state `default`
+            // If the desire is to have additional states you can add logic here to build
+            // up a list and iterate through each state in the list using the loop below.
+            // This must be done after potentially loading states from an external file.
             var stateNames = new List<String>();
-            if (!string.IsNullOrEmpty(cmdLineArgs.PresetStateID))
-            {
-                if (cmdLineArgs.PresetStateID == "all")
-                {
-                    foreach (var state in model.CurrentWorkspace.Presets)
-                    {
-                        stateNames.Add(state.Name);
-                    }
-                }
-                else
-                {
-                    stateNames.Add(cmdLineArgs.PresetStateID);
-                }
-            }
-            else
-            {
-                stateNames.Add("default");
-            }
+            stateNames.Add("default");
 
```

Comment:
With these changes the `default` state isn't actually used for anything other than populating the list of `stateNames` with a single item ensuring the graph executes 1 time.  I left the `foreach` loop below to show how additional logic could be implemented to execute additional states if desired now that presets have been removed.